make/autoconf/otp.m4

```diff
@@ -3027,6 +3029,10 @@ AC_DEFUN([LM_HARDWARE_ARCH], [
 	AC_MSG_RESULT(yes: adjusting ARCH=arm to ARCH=arm64)
 	ARCH=arm64
 	;;
+    riscv64)
```

Comment:
This will never match, since the case values are on the form <arch>-<4-or-8>. Also it's not clear to me that RISC-V needs this kludge. Are there RISC-V systems that report themselves as riscv64 but generate 32-bit code by default, or report themselves as riscv32 but generate 64-bit code by default? If not, then you don't need to touch this bit.

The origin for was operating systems like Solaris that ran on 64-bit hardware but defaulted (at the time at least) to generating 32-bit code.

Reply:
Thanks for the code review @mikpe, I'll update the patch to follow your code review! :)